packages/serverpod/lib/src/server/websocket_request_handlers/endpoint_websocket_request_handler.dart

```diff
@@ -92,7 +92,7 @@ abstract class EndpointWebsocketRequestHandler {
                   .deserializeByClassName(serialization);
 
               if (message == null) throw Exception('Streamed message was null');
-
+              // OLD WAY
```

Comment:
Suggestion: I don't think this comments add any value, I would remove it.

Reply:
Outdated code, I think it was in an earlier commit.

Reply:
Reverted here: [cec5fad](https://github.com/serverpod/serverpod/pull/2676/commits/cec5fad3d1dececc42a205ca6662da6c4342f464)